openc3/lib/openc3/interfaces/stream_interface.rb

```diff
@@ -40,8 +40,8 @@ def initialize(protocol_type = nil, protocol_args = [])
     end
 
     def connect
-      super()
       @stream.connect if @stream
+      super()
```

Comment:
Sending the command in the `connect` method only works if we do the `super()` last.